platform-emf/src/main/java/com/softicar/platform/emf/data/table/EmfDataTableRow.java

```diff
@@ -125,33 +128,50 @@ private <T> void setDefaultTextAlign(IEmfDataTableColumn<R, T> column, EmfDataTa
 		cell.setStyle(cell.getColumn().getSettings().getAlignmentOrDefault(textAlign));
 	}
 
-	private class ActionCell extends DomCell implements IEmfDataTableActionCell<R> {
+	private class ActionCell extends DomDelegatingParentElement implements IEmfDataTableActionCell<R> {
```

Comment:
- is it really necessary to use `DomDelegatingParentElement` here?
- wouldn't it be much easier to drop the `container` and fix it all with CSS?

Reply:
Without the container, we'd need to directly style the `td` element. That is, we'd need to align children next to each other, with a gap. `flex` or `grid` are the only reasonable means I know to achieve this (`padding` or `margin` are not viable, IMO).
Any suggestions?

Reply:
ok, seems there is currently no better solution